src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -1757,17 +1757,23 @@ bool WindowImplX11::processEvent(XEvent windowEvent)
         // Key down event
         case KeyPress:
         {
-            // Get the keysym of the key that has been pressed
-            static XComposeStatus keyboard;
-            char buffer[32];
-            KeySym symbol;
-            XLookupString(&windowEvent.xkey, buffer, sizeof(buffer), &symbol, &keyboard);
+            Keyboard::Key key = Keyboard::Unknown;
+
+            // Try each KeySym index (modifier group) until we get a match
+            for (int i = 0; i < 4; ++i)
```

Comment:
I see, so on Linux, this can basically be Shift, Control, Super, Meta, Hyper, Mod1, Mod2, Mod3, Mod4, Mod5. :P Thanks for clarification.